.github/CODEOWNERS

```diff
@@ -16,6 +16,8 @@
 /typings/ @ankita240796
 /tsconfig.json @ankita240796
 /tsconfig-for-compile-check.json @ankita240796
+/core/templates/dev/head/services/UpgradedServices.ts @bansalnitish
+/core/templates/dev/head/services/contextual/LoggerService.ts @bansalnitish
```

Comment:
Move this to after line 15, we want to keep this in alphabetical order.

Reply:
Done